ghost/core/core/server/data/schema/fixtures/fixtures.json

```diff
@@ -623,6 +623,11 @@
                     "name": "Browse links",
                     "action_type": "browse",
                     "object_type": "link"
+                },
+                {
+                    "name": "Bulk edit links",
+                    "action_type": "bulkEdit",
```

Comment:
Should this just be an `edit` permission?

Reply:
🤔 Yea i was thinking of action type used in endpoint, but maybe makes more sense to update this to `edit` and update permissions for endpoint to use that.

Reply:
Updated to `edit`